### PR TITLE
[FR] Update "Beatmap discussion"

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page "Beatmap discussion".

- Addition of the French images
- Update `mode-selection.jpg` because it was not up to date (I modified the redirections on `en.md` and `nl.md`).

### CODE REVIEW

File: wiki/Beatmap_Discussion/fr.md [hunk 1]
```diff
@@ -1,228 +1,164 @@
...
+## En-tête de la beatmap
 
-![](img/info.jpg "Page d'information Beatmap")
+*Pour plus d'informations sur le paramétrage des métadonnées, voir : [Song Setup § Song and Map Metadata](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata)*
 
-### Grâce à osu.ppy.sh
+![](img/beatmap-header-FR.png "En-tête de la beatmap")
 
-Cliquer sur `Modding/Discussion` sur la page de la beatmap vous amènera à la page, si elle est actuellement dans le système de discussion du Beatmap.
+L'en-tête du beatmap affiche le titre et l'artiste fournis dans l'[éditeur du beatmap](/wiki/Beatmap_Editor). En cliquant sur l'en-tête du beatmap, vous retournerez à la page d'information.
```
Comment (Satsukel) [inline wiki/Beatmap_Discussion/fr.md:48]:
```suggestion
L'en-tête de la beatmap affiche le titre et l'artiste fournis dans l'[éditeur du beatmap](/wiki/Beatmap_Editor). En cliquant sur l'en-tête de la beatmap, vous retournerez à la page d'information.
```

File: wiki/Beatmap_Discussion/fr.md [hunk 2]
```diff
@@ -1,228 +1,164 @@
...
+## Visualisation de la timeline
 
-### Statut de modification
+![](img/visual-timeline-FR.png "Posts de modding répartis sur la timeline")
 
-![](img/modding-status.jpg "Pour vos besoins de tri")
+La visualisation de la timeline affiche tous les mods réalisés pour une difficulté qui utilisent un horodatage. En cliquant sur l'une d'entre elles, vous ferez défiler la liste jusqu'au mod posté à cette date. Notez que toutes les options de filtrage modifieront ce qui est affiché ici. La visualisation de la timeline est un outil utile pour voir d'un coup d'œil combien une beatmap a été modifiée. Une timeline serrée signifie qu'un beatmap a déjà reçu beaucoup d'attention. Les paramètres de la difficulté sélectionnée sont également affichés en haut à droite de la visualisation de la timeline. De gauche à droite, ils décrivent la `Longueur`, le `BPM`, le `Circle Count` et le `Slider Count` d'une difficulté.
```
Comment (Satsukel) [inline wiki/Beatmap_Discussion/fr.md:73]:
```suggestion
La visualisation de la timeline affiche tous les mods réalisés pour une difficulté qui utilisent un horodatage. En cliquant sur l'une d'entre elles, vous ferez défiler la liste jusqu'au mod posté à cette date. Notez que toutes les options de filtrage modifieront ce qui est affiché ici. La visualisation de la timeline est un outil utile pour voir d'un coup d'œil à quel point une beatmap a été modifiée. Une timeline serrée signifie qu'une beatmap a déjà reçu beaucoup d'attention. Les paramètres de la difficulté sélectionnée sont également affichés en haut à droite de la visualisation de la timeline. De gauche à droite, ils décrivent la `Longueur`, le `BPM`, le `Circle Count` et le `Slider Count` d'une difficulté.
```

File: wiki/Beatmap_Discussion/fr.md [hunk 3]
```diff
@@ -1,228 +1,164 @@
...
+### Train de la hype
 
-## Statut de Beatmap
+Le train de la hype permet de savoir combien de [hypes](/wiki/Beatmap/Hype) un mapset a reçu. Une fois que 5 hypes ont été atteints, la beatmap peut être nommé par les [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators). Les hypes ne peuvent être données qu'en allant dans l'onglet `Général (Toutes les difficultés)`, car cela ne fonctionnera pas à travers les onglets `Général` ou `Chronologie`. En cliquant sur le bouton "Hype", vous serez automatiquement dirigé vers le bon onglet.
 
-![](img/beatmap-status.jpg "Contient des informations importantes !")
+### Statut de la nomination
 
-### Créateur de Beatmap
+Le statut de la nomination permet de suivre les nominations d'un beatmap. Lorsqu'une beatmap a reçu deux nominations, elle est qualifié.
```
Comment (Satsukel) [inline wiki/Beatmap_Discussion/fr.md:92]:
```suggestion
Le statut de la nomination permet de suivre les nominations d'une beatmap. Lorsqu'une beatmap a reçu deux nominations, elle est qualifié.
```

File: wiki/Beatmap_Discussion/fr.md [hunk 4]
```diff
@@ -1,228 +1,164 @@
...
+### Suivre/Ne plus suivre
 
-### Hype Train
+Les boutons `Suivre` et `Ne plus suivre` permettent de suivre/ne plus suivre les développements d'une beatmap. Si une beatmap est suivi, les nouveaux messages et les réponses sur la page de discussion de cette beatmap créeront des notifications sur osu!web. Les abonnements peuvent être gérés par le biais de la [liste de suivi](https://osu.ppy.sh/beatmapsets/watches), qui se trouve dans le menu de la page d'accueil.
 
-*Note : Vous devez maintenant cliquer sur `Hype` pour ajouter du hype ; donner des éloges n'ajoutera plus de hype automatiquement.*
+### Page de la beatmap
 
-Le train Hype est similaire au précédent système de priorité des étoiles.
+Le bouton `Page de la beatmap` permet de naviguer vers la page d'information d'un beatmap. Cela peut également être fait en cliquant sur l'[en-tête de la beatmap](#en-tête-de-la-beatmap).
```
Comment (Satsukel) [inline wiki/Beatmap_Discussion/fr.md:104]:
```suggestion
Le bouton `Page de la beatmap` permet de naviguer vers la page d'information d'une beatmap. Cela peut également être fait en cliquant sur l'[en-tête de la beatmap](#en-tête-de-la-beatmap).
```

File: wiki/Beatmap_Discussion/fr.md [hunk 5]
```diff
@@ -1,228 +1,164 @@
...
+### Chronologie des discussions
 
-L'activation de la broche permettra au [champ de modification](#modding-field) pour défiler avec vous, une fois que vous avez dépassé le champ de modification.
+La timeline à gauche d'une discussion montre l'horodatage du message. Ceci est seulement disponible dans la section `Chronologie`. Sinon, la timeline de la discussion sera laissée vide.
 
-## Mod Champ de discussion
+### Tags
 
-![](img/discussions.jpg)
+Les tags affichent les [rôles importants](/wiki/People/The_Team) sous le nom de l'utilisateur. Seuls les rôles pertinents pour la discussion sur le beatmap sont affichés. Cela inclut les membres `MAPPER`, `BN`, `NAT`, `GMT`, et `DEV` qui ont plus d'options qu'un utilisateur normal regardant une page de discussion.
```
Comment (Satsukel) [inline wiki/Beatmap_Discussion/fr.md:152]:
```suggestion
Les tags affichent les [rôles importants](/wiki/People/The_Team) sous le nom de l'utilisateur. Seuls les rôles pertinents pour la discussion sur la beatmap sont affichés. Cela inclut les membres `MAPPER`, `BN`, `NAT`, `GMT`, et `DEV` qui ont plus d'options qu'un utilisateur normal regardant une page de discussion.
```

**Review by Satsukel — APPROVED** [review]

**Review by Satsukel — APPROVED** [review]

**Review by TicClick — APPROVED** [review]
merging with a couple of fixes here and there:
- please always keep the original article's tags
- there really is no point in duplicating images that have no localized text
- changing the extension doesn't convert the image:
  ```bash
  blah% file wiki/Beatmap_Discussion/img/visual-timeline-FR.png
  wiki/Beatmap_Discussion/img/visual-timeline-FR.png: JPEG image data, baseline, precision 8, 934x164, components 3
  ```

**Pull request merged**